app/Http/Controllers/BeatmapPacksController.php

```diff
@@ -55,4 +67,14 @@ public function show($id)
 
         return view('packs.show', compact('pack', 'sets', 'mode'));
     }
+
+    private function indexLink(BeatmapPack $pack) : string
+    {
+        $type = $pack->type();
+        $indexInPagination = BeatmapPack::getPacks($type)->get()->search($pack);
```

Comment:
Not sure if want to do this. It doesn't expand either. Might as well just display expanded single item.